device-identifier/shbdeviceidentifier/utilities/queries.py

```diff
@@ -2,7 +2,7 @@
 
 from .app_utilities import IDENTIFIER_HOME
 
-q_path = IDENTIFIER_HOME / Path("utilities/queries").resolve()
```

Comment:
I'd prefer 

``` python
q_path = IDENTIFIER_HOME / Path("utilities/queries")
```

since .resolve() is most likely the problem.

Reply:
Updated.

Reply:
We still need Path() for "utilities/queries", don't we?